authzed/api/v1/error_reason.proto

```diff
@@ -220,4 +220,19 @@ enum ErrorReason {
   //       }
   //     }
   ERROR_REASON_INVALID_CURSOR = 15;
+
+  // The request failed because there are too many matching relationships to be
+  // deleted within a single transactional deletion call. To avoid, set
+  // `optional_allow_partial_deletions` to true on the DeleteRelationships call.
+  //
+  // Example of an ErrorInfo:
+  //
+  //     {  
+  //       "reason": "ERROR_REASON_TOO_MANY_RELATIONSHIPS_FOR_TRANSACTIONAL_DELETE",
+  //       "domain": "authzed.com",
+  //       "metadata": {
+  //          ... fields for the filter ...
+  //       }
+  //     }
+  ERROR_REASON_TOO_MANY_RELATIONSHIPS_FOR_TRANSACTIONAL_DELETE = 16;
```

Comment:
is it up to the server to define what "too many" is? will there be a CLI option?

Reply:
No; its specified via the limit on the delete relationships call

Reply:
so it it exceeds the number of relationships to delete, it will return this error?